python/mlir_structured/runtime/util.py

```diff
@@ -0,0 +1,21 @@
+import contextlib
+from typing import Optional
+
+from mlir_structured.ir import Context, Module, InsertionPoint, Location
+
+
+@contextlib.contextmanager
+def mlir_mod_ctx(src: Optional[str] = None,
+                 context: Context = None,
+                 location: Location = None):
```

Comment:
Nice! That looks useful -- thanks!

I am wondering whether this isn't a testing util (and the file should, thus, live under `test/`)...?